stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -182,9 +181,8 @@ public void prepare() throws Exception {
       throw new IllegalStateException("Some job worker crashed or joined after data are written. "
           + "The test is stopped.");
     }
-    mJobWorkerDirNames = Arrays.asList(jobWorkerDirs).stream()
-        .map(file -> file.getName())
-        .collect(Collectors.toList());
+    mJobWorkerDirNames =
```

Comment:
if it is immutable, perhaps use `tounmodifiablelist`

Reply:
singletonList is immutable